docs/development-tips.md

```diff
@@ -30,9 +30,10 @@ The non-v1 apps are the following:
  - know-before-you-owe (ghe/CFGOV/know-before-you-owe) - for the Consumer Tools > Know before you owe section.
 
 After installing these projects as sibling directories to the `cfgov-refresh` repository,
-build the third-party projects per their directions,
-stop the web server and return to `cfgov-refresh`
-and run `cfgov/manage.py sheer_index -r` to load the projects' data into ElasticSearch.
```

Comment:
This is still necessary for owning-a-home, I believe. 

It might make sense to instead add a note below the replacement text above sheer-based apps (owning-a-home is public but there are a couple others that are not).